GetandTake/Services/AutoMapper/AutoMapperProfile.cs

```diff
@@ -0,0 +1,20 @@
+﻿using AutoMapper;
+using GetandTake.Models;
+using GetandTake.Models.DTOs.BaseDTO;
+using GetandTake.Models.DTOs.ListDTO;
+
+namespace GetandTake.Services.AutoMapper
+{
+    public class AutoMapperProfile : Profile
+    {
+        public AutoMapperProfile()
+        {
+            CreateMap<Product, ProductDTO>().ReverseMap();
+            CreateMap<Product, ProductsDTO>()
```

Comment:
Let's add some formatting to your methods to make it easier and better to read:
```suggestion
            CreateMap<Product, ProductsDTO>()
                   .ForMember(dto => dto.Category, 
                                           entity => entity.MapFrom(entity => entity.Category.CategoryName))
                   .ForMember(dto => dto.Supplier,
                                           entity => entity.MapFrom(entity => entity.Supplier.CompanyName));
```

In general, if you have a chain of methods call, LINQ example, it's better to put all these methods into column with same alignment, it really simplifies reading